pybossa/view/bloomberg.py

```diff
@@ -104,3 +109,14 @@ def handle_bloomberg_response():
         current_app.logger.exception('BSSO login error')
         flash(gettext('We were unable authenticate and log you into an account. Please contact a Gigwork administrator.'), 'error')
         return redirect(url_for('home.home'))
+
+
+def get_user_data_access_level(firm_num):
+    """Reads firm id to user type mappings from settings_upref_mdata and
+    returns the access type"""
+    firm_num_to_type = current_app.config.get('FIRM_TO_TYPE')
+    if current_app.config.get('PRIVATE_INSTANCE') and firm_num:
```

Comment:
A minor one, we used to avoid using "PRIVATE_INSTANCE" keyword as this is a public repository, could consider using "VALID_ACCESS_LEVELS_FOR_USER_TYPES" to indicate that this is not public instance.